mathics/builtin/numpy_utils/with_numpy.py

```diff
@@ -330,7 +330,10 @@ def transform(self):
 
         tree = self.visit(tree)
         tree = ast.fix_missing_locations(tree)
-        tree.posonlyargs = []
+        
+        for node in tree.body:
+            node.args.posonlyargs = []
```

Comment:
I believe node.args already has a posonlyargs field when in Python3.8+ (At least that's what my debugger told me when I last looked a while ago.)

Reply:
So why is `compile` raising an error?